src/main/java/uk/gov/hmcts/reform/orgrolemapping/controller/WelcomeController.java

```diff
@@ -1,93 +1,24 @@
-import uk.gov.hmcts.reform.orgrolemapping.domain.model.enums.UserType;
-import uk.gov.hmcts.reform.orgrolemapping.domain.service.BulkAssignmentOrchestrator;
-import uk.gov.hmcts.reform.orgrolemapping.v1.V1;
-
-import static org.springdoc.core.Constants.SWAGGER_UI_URL;
-import static uk.gov.hmcts.reform.orgrolemapping.apihelper.Constants.AUTHORIZATION;
-import static uk.gov.hmcts.reform.orgrolemapping.apihelper.Constants.SERVICE_AUTHORIZATION;
 
 @RestController
 @Slf4j
 @NoArgsConstructor
 @Hidden
 public class WelcomeController {
 
-    private BulkAssignmentOrchestrator bulkAssignmentOrchestrator;
-
-
-    @Autowired
-    public WelcomeController(
-            BulkAssignmentOrchestrator bulkAssignmentOrchestrator) {
-        this.bulkAssignmentOrchestrator = bulkAssignmentOrchestrator;
-    }
-
-    @GetMapping(value = "/swagger")
-    public RedirectView swaggerRedirect() {
-        return new RedirectView(SWAGGER_UI_URL, true, false);
-    }
-
+    //This is just a test API
     @GetMapping(value = "/welcome")
     public String welcome() {
         return "Welcome to Organisation Role Mapping Service";
     }
-    //This is just a test API
-
-    @PostMapping(
-            path = "/am/role-mapping/staff/users",
-            produces = V1.MediaType.MAP_ASSIGNMENTS,
-            consumes = {"application/json"}
-    )
-    @ResponseStatus(code = HttpStatus.OK)
-    @Operation(summary = "creates multiple role assignments based upon user profile mapping rules",
-            security =
-            {
-                @SecurityRequirement(name = AUTHORIZATION),
-                @SecurityRequirement(name = SERVICE_AUTHORIZATION)
-            })
-    @ApiResponse(
-            responseCode = "200",
-            description = "OK",
-            content = @Content(schema = @Schema(implementation = Object.class))
-    )
-    @ApiResponse(
-            responseCode = "400",
-            description = V1.Error.INVALID_REQUEST,
-            content = @Content()
-    )
-    public ResponseEntity<Object> createOrgMapping(@RequestBody UserRequest userRequest,
-                                                   @RequestHeader(value = "userType")
-                                                           UserType userType) {
-        var startTime = System.currentTimeMillis();
-        log.debug("createOrgMapping");
-        log.info("Process has been Started for the userIds {}", userRequest.getUserIds());
-        ResponseEntity<Object> response = bulkAssignmentOrchestrator.createBulkAssignmentsRequest(userRequest,
-                userType);
-        log.debug("Execution time of createOrgMapping() : {} ms",
-                (Math.subtractExact(System.currentTimeMillis(), startTime)));
-        return ResponseEntity.status(response.getStatusCode()).body(response.getBody());
-    }
-
 
     //This method needed for the functional tests, so that RAS gets enough time to create records.
     @PostMapping(value = "/sleep")
```

Comment:
This API should probably be retired as there are sleep options built into the BEFTA framework that would suit the functional test this better